CameraEngine/CameraEngine.swift

```diff
@@ -411,10 +411,18 @@ public extension CameraEngine {
                 do {
                     try currentDevice.lockForConfiguration()
                     currentDevice.focusPointOfInterest = CGPoint(x: focusPoint.x, y: focusPoint.y)
-                    currentDevice.focusMode = AVCaptureFocusMode.AutoFocus
+                    if currentDevice.focusMode == AVCaptureFocusMode.Locked {
+                        currentDevice.focusMode = AVCaptureFocusMode.AutoFocus
+                    } else {
+                        currentDevice.focusMode = AVCaptureFocusMode.ContinuousAutoFocus
```

Comment:
One strange thing I noticed in AVFoundation is that you **have to** explicitly call `currentDevice.focusMode = AVCaptureFocusMode.ContinuousAutoFocus` even if is already in `.ContinuousAutoFocus`.

Any explanation for that ? 😶